Levels/SampleBase/SampleBase.prefab

```diff
@@ -163,11 +165,11 @@
                 "Component_[12802329719955739455]": {
                     "$type": "EditorScriptCanvasComponent",
                     "Id": 12802329719955739455,
-                    "m_name": "SpawnIfAuthority",
-                    "runtimeDataIsValid": true,
-                    "sourceHandle": {
-                        "id": "{B605AD71-0689-5650-B3F5-558D471B6351}",
-                        "path": "C:/Git/Spectra/o3de/MultiplayerSample/scriptcanvas/SpawnIfAuthority.scriptcanvas"
+                    "configuration": {
+                        "sourceHandle": {
+                            "id": "{B605AD71-0689-5650-B3F5-558D471B6351}",
+                            "path": "C:/Git/Spectra/o3de/MultiplayerSample/scriptcanvas/SpawnIfAuthority.scriptcanvas"
```

Comment:
Shouldn't this path be related?

Reply:
I remember creating a GHI for this month back for scripting. I recall it not mattering, since asset id is what's really used. That isn't even the path on my hard drive.

Reply:
@carlitosan, is this normal or should I create a GHI?

Reply:
It has been fixed, merge latest and re-save, or manually modify the content. It shouldn't ever save as absolute again.

Reply:
@carlitosan After grabbing latest the path is still absolute, except now it points to the location on my hard drive instead of the previous user.

Reply:
I can create a new GHI if this is a bug. That shouldn't hold up this PR though, right? Everything will still work for users that pull down the new change?

Reply:
I'm running development commit 61ec460ab1efc5ae6e7c024c008e72d2c71d1c7d Tue May 3 09:29:00 -0700

Reply:
It is using a relative path for the ID as that hasn't changed with re-saving the file.
If the "path" is only a hint, then this isn't an issue.
If the path is actually used, then that is is a problem.